pkg/users/account-create-dialog.js

```diff
@@ -62,12 +80,32 @@
                            value={user_name} onChange={value => change("user_name", value)} />
             </FormGroup>
 
-            <PasswordFormFields password_label={_("Password")}
-                                password_confirm_label={_("Confirm")}
-                                error_password={errors?.password}
-                                error_password_confirm={errors?.password_confirm}
-                                idPrefix="accounts-create-password"
-                                change={change} />
+            <FormGroup label={_("Home directory")}
+                       helperTextInvalid={errors && errors.home_dir}
+                       validated={(errors && errors.home_dir) ? "error" : "default"}
+                       fieldId="accounts-create-user-home-dir">
+                <TextInput id="accounts-create-user-home-dir"
+                    onChange={value => change("home_dir", value)}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18234-20230317-103941-df531fdc-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)